Fixing permissions for external /data volumes
Using bin/standalone
Sending logs to graylog: 'logcentral.eea.europa.eu:12201' as facilty: 'eea.docker.plone'

[buildout]
extends = develop.cfg
find-links += 
develop += src/eea.facetednavigation
eggs += eea.facetednavigation
zcml += eea.facetednavigation-meta

[plonesite]
enabled = False
site-id = Plone
profiles += eea.facetednavigation:default

[versions]
